docs/ref/templates/builtins.txt

```diff
@@ -1436,6 +1436,39 @@ then the output would be::
 
 .. templatefilter:: dictsortreversed
 
+While ``dictsort`` was designed to work with list of dictionaries, it is also
+possible to order a list of lists (or any other object, implementing
+``__getitem__``) by elements at specified index.
+For example, the following call to ``dictsort``::
+
+    {% vars|dictsort:0 %}
+
+Where ``vars`` is defined as:
+
+.. code-block:: python
+
+    [
+        ('a', '42'),
+        ('c', 'string'),
+        ('b', 'foo')
+    ]
+
+produces the following output:
+
+.. code-block:: python
+
+    [
+        ('a', '42'),
+        ('b', 'foo'),
+        ('c', 'string')
+    ]
+
+Note, that you should pass index as an integer, rather than a string. The following block::
+
+    {% vars|dictsort:"0" %}
+
+produces empty output.
+
```

Comment:
Add a `.. versionchanged:: 1.10` annotation with a description of what changed, as well as a mention in the 1.10 release notes.